File: backend/slotify/spring/src/main/java/com/slotify/backend/spring/company/dtos/UpdateIntervalResponse.java
package com.slotify.backend.spring.company.dtos;

import lombok.Builder;
import lombok.Getter;
import lombok.ToString;

import java.time.LocalDateTime;
import java.util.UUID;

@Builder
@Getter
@ToString
public class UpdateIntervalResponse {

    private UUID intervalId;
    private LocalDateTime startDatetime;
    private LocalDateTime endDatetime;
    private Integer maxConcurrentServices;
    private LocalDateTime createdAt;
}

File: frontend/app/src/main/java/com/raj/slotify/dtos/company/UpdateIntervalResponse.kt
package com.raj.slotify.dtos.company

import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime
import java.util.UUID

data class UpdateIntervalResponse(

    @SerializedName("intervalId")
    val intervalId: UUID,

    @SerializedName("startDatetime")
    val startDateTime: LocalDateTime,

    @SerializedName("endDatetime")
    val endDateTime: LocalDateTime,

    @SerializedName("maxConcurrentServices")
    val maxConcurrentServices: Int,

    @SerializedName("createdAt")
    val createdAt: LocalDateTime

)
